docs/admin/cold-starts.md

```diff
@@ -0,0 +1,14 @@
+# Improve cold starts 
+
+Cloud Run comes with a number of settings that can improve performance around cold starts. These
+aren't set by default, as these settings can increase the running costs of the service. 
+
+To prevent cold starts, Cloud Run supports setting [minimum instances](https://cloud.google.com/run/docs/configuring/min-instances). If the minimum instance is set to "1", Cloud Run will always be running, and never start cold, thus eliminating this performance issue. Check the [Cloud Run pricing](https://cloud.google.com/run/pricing) before configuring this setting.  
+
+In general, there are a number of configurations that can be customized to improve performance: 
```

Comment:
feels like a missing set of guidance around 

* monitoring performance (so you know when you might need to care about this) 
* container management (does size of container matter, does language matter?

Reply:
Added a link to the general tips docs instead of repeating a lot of nuance here. 

The general aim of this doc was summarising the cameo from Next.